app/controllers/Support.scala

```diff
@@ -0,0 +1,43 @@
+package controllers
+
+import play.api.libs.ws.WSClient
+import play.api.mvc.Controller
+import com.squareup.okhttp.{Credentials, OkHttpClient, Request}
+import config.Config
+import java.util.concurrent.TimeUnit
+
+
+
+class Support(val wsClient: WSClient) extends Controller with PanDomainAuthActions {
+
+  def previewCapiProxy(path: String) = APIAuthAction { request =>
+    val httpClient = new OkHttpClient()
```

Comment:
Are you sure you want to create a new client each time this function is called? Really it should be a member of the class, or wrapped by an object and used as a singleton.